install.py

```diff
@@ -5,17 +5,19 @@
 from multiprocessing import Pool
 
 
-def build(library, args):
+def build(library, args, dockerfile):
```

Comment:
Isn't dockerfile identical to args.algorithm? (Maybe the latter has a confusing name?) Or do you really want to point to dockerfile that are not present in `install/`?

Reply:
I think my rationale was to keep T3 docker builds out of the default when you run "python install.py."  Is there a better way to accomplish this?

Reply:
I see, let's keep it like you proposed.